src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from typing import Any, Mapping
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.group import GroupEndpoint
+from sentry.eventstore.models import GroupEvent
+from sentry.seer.utils import SimilarIssuesEmbeddingsRequest, get_similar_issues_embeddings
+from sentry.web.helpers import render_to_string
+
+logger = logging.getLogger(__name__)
+
+
+def get_stacktrace_string(exception: Mapping[Any, Any], event: GroupEvent) -> str:
```

Comment:
The exception string is currently formatted like the stacktraces in the [poc spreadsheet](https://docs.google.com/spreadsheets/d/1menVeYMQCxgE4mha8if4XRuM2PXfQxJsoo9F_7-eQTU/edit#gid=55932162)